catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/config/ConfigurationApplication.java

```diff
@@ -397,8 +399,19 @@ private void setProxiesForImagery(List<Map<String, Object>> newImageryProviders)
         proxiedImageryProviders.clear();
         for (Map<String, Object> newImageryProvider : newImageryProviders) {
             HashMap<String, Object> map = new HashMap<>(newImageryProvider);
-            map.put(URL, SERVLET_PATH + "/" + urlToProxyMap.get(newImageryProvider.get(URL)
-                    .toString()));
+            String baseUrl = newImageryProvider.get(URL).toString();
+            boolean proxyEnabled = true;
+            Object proxyEnabledProp = newImageryProvider.get(PROXY_ENABLED);
+            if (proxyEnabledProp instanceof String) {
+                proxyEnabled = Boolean.parseBoolean((String)proxyEnabledProp);
+            }
+
+            if (proxyEnabled) {
+                map.put(URL,
+                        SERVLET_PATH + "/" + urlToProxyMap.get(baseUrl));
+            } else {
+                map.put(URL, baseUrl);
+            }
             proxiedImageryProviders.add(map);
```

Comment:
It's really the original URL. I will update the variable names. Unfortunately they were already using imageryProviders for the incoming JSON, so I'll pick another.